README.md

```diff
@@ -88,22 +97,36 @@ Please be respectful of the people who have created these resources. If you want
 * [Chemics](https://github.com/wigging/chemics) - Python package for chemistry and chemical engineering applications.
+* [ht](https://github.com/CalebBell/ht) - Python package for heat transfer in chemical engineering design.
+* [fluids](https://github.com/CalebBell/fluids) - Python package for fluid dynamics in chemical engineering design
+* [thermo](https://github.com/CalebBell/thermo) - Python package for thermodynamics and phase equilibrium in chemical engineering design.
 * [Clapeyron.jl](https://github.com/ClapeyronThermo/Clapeyron.jl) - Julia package for thermodynamic property calculations / equations of state.
 * [OpSim](https://github.com/opsim/opsim) - open source Chemical Engineering Process Simulator with a user friendly drag-and-drop graphical user interface and an underlying high performance simulation engine.
 * [Process Dynamics Engine](https://github.com/OpenChemE/Process-Dynamics-Engine) - online, real-time simulator for process control models described by transfer functions or state space representations.
 * [pychemqt](https://github.com/jjgomera/pychemqt) - free software tool for calculation and design of unit operations in chemical engineering.
 * [Reaction Mechanism Generator (RMG)](https://github.com/ReactionMechanismGenerator) - automatic chemical reaction mechanism generation.
+* [Psychrochart](https://github.com/azogue/psychrochart) - interactive psychrometric charts and overlay information
+* [pyflowsheet](https://github.com/Nukleon84/pyflowsheet) - python package for drawing process flow diagrams
+* [reaction-network](https://github.com/materialsproject/reaction-network) - Python package for predicting likely inorganic chemical reaction pathways using graph theoretical methods
+* [ASALI](https://github.com/srebughini/ASALI) - reactor modeling
+* [Cantera](https://cantera.org/) - tools for problems involving chemical kinetics, thermodynamics, and transport processes
+* [Summit](https://github.com/sustainable-processes/summit) - Optimising chemical reactions using machine learning
+* [Peroxide](https://github.com/imnotdev25/Peroxide) - Rust numeric lib similar to numpy
+* [Reactorch](https://github.com/DENG-MIT/reactorch) - pytorch package for differentiable reacting flow simulation
+* [pyMARS](https://github.com/Niemeyer-Research-Group/pyMARS) - python package for chemical kinetic model
 
 ## Journals
 * [Chemical Engineering Education](https://journals.flvc.org/cee)
     * [Fall 2022 Special Issue - Teaching Data Science](https://journals.flvc.org/cee/issue/view/6050)
     * [Summer 2020 Special Issue - CACHE 50th Anniversary](https://journals.flvc.org/cee/issue/view/5698)
 * [Education for Chemical Engineers](https://www.sciencedirect.com/journal/education-for-chemical-engineers)
 * [Journal of Chemical Education](https://pubs.acs.org/journal/jceda8)
+* [Computers & Chemical Engineering](https://www.sciencedirect.com/journal/computers-and-chemical-engineering)
```

Comment:
these should only be journals related to chemical engineering *education*, so I don't think this belongs